umap/static/umap/js/umap.features.js

```diff
@@ -1,25 +1,70 @@
 U.FeatureMixin = {
   staticOptions: { mainColor: 'color' },
 
-  initialize: function (map, latlng, options) {
+  getSyncMetadata: function () {
+    return {
+      engine: this.map.sync,
+      subject: 'feature',
+      metadata: {
+        id: this.id,
+        layerId: this.datalayer?.id || null,
+        featureType: this.getClassName(),
+      },
+    }
+  },
+
+  onCommit: function () {
+    // When the layer is a remote layer, we don't want to sync the creation of the
+    // points via the websocket, as the other peers will get them themselves.
+    if (this.datalayer.isRemoteLayer()) return
+    const { subject, metadata, engine } = this.getSyncMetadata()
+    engine.upsert(subject, metadata, this.toGeoJSON())
+  },
+
+  getGeometry: function () {
+    return this.toGeoJSON().geometry
+  },
+
+  syncUpdatedProperties: function (properties) {
+    // When updating latlng, sync the whole geometry
+    if ('latlng'.includes(properties)) {
```

Comment:
it's basically the same thing as `'latlng' in properties` (in python). Yeah, JavaScript.

Reply:
I'm still bugging :p

![image](https://github.com/umap-project/umap/assets/146023/7d0fab94-0bed-4de4-ba4c-26b8c062d4e2)

I would expect something like this instead, but I may still missing the point: `properties.includes("latlng")`

Reply:
😲 Actually, that was a mistake on my part. What I wanted to do is indeed `properties.includes('latlng')`.

Because this `syncUpdatedProperties` is only called once, it didn't trigger any issue in practice (as the check was always returning `true`). Thanks for spotting this 👍

Reply:
Fixed in 846901a8af34ce7e8430dc8f8c7a434bb5da77e6